src/components/powerHour/PowerHourTimer.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { Player } from "@lottiefiles/react-lottie-player";
+import { PauseIcon, PlayIcon, ResetIcon } from "../icons/Icons";
+
+const ONE_HOUR_IN_SECONDS = 3600;
+const formatTime = (timeVal) => {
+  if (timeVal <= 9) {
+    return `${0}${timeVal}`;
+  }
+  return timeVal;
+};
+
+export const PowerHourTimer = () => {
+  const confettiTop = React.useRef();
+  const confettiBottom = React.useRef();
+  const [paused, setPaused] = React.useState(true);
+  const [secondsLeft, setSecondsLeft] = React.useState(ONE_HOUR_IN_SECONDS);
+
+  React.useEffect(() => {
+    // if the timer is paused no need to continue counting
+    const interval = !paused
+      ? setInterval(() => {
+          setSecondsLeft((secondsLeft) => {
+            if (secondsLeft <= 0) {
+              return 0;
+            }
+
+            return secondsLeft - 1;
+          });
+        }, 1000)
+      : null;
+
+    // cleanup the interval on unload
+    return () => clearInterval(interval);
+  }, [paused]);
+
+  const timerParts = [
+    // hours
+    Math.floor(secondsLeft / ONE_HOUR_IN_SECONDS),
+    // minutes
+    Math.floor((secondsLeft % ONE_HOUR_IN_SECONDS) / 60),
+    // seconds
+    Math.floor(secondsLeft % 60),
+  ];
+
+  const seconds = timerParts[2];
+
+  if (seconds === 0 && timerParts[0] === 0) {
+    // fire the animation and sound
+    const shots = new Audio("/audio/shots.mp3");
```

Comment:
I think the sound should be a bell, or a ding. If we leave it as shots we add liability to the website lol as promoting drinking. Don't want to fly too close to the sun there.

Reply:
Yeah totally sounds good to me